extensions/objects/templates/math-editor.directive.html

```diff
@@ -0,0 +1,3 @@
+<div class="guppy-div"
+     style="height: 100px; padding: 5px; overflow: auto; word-break: normal; word-wrap: normal;">
```

Comment:
Create a class for the styling used here.

Reply:
Done.